integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -710,11 +688,8 @@ func nodeEndGame(
 ) {
 
 	fmt.Println("Calling SC.endGame...")
-	txScCall := createTxEndGame(nodes[idxNode], round, scAddress)
+	txScCall := createTxEndGame(nodes[idxNode].OwnAccount, round, scAddress)
 	nodes[idxNode].SendTransaction(txScCall)
-	time.Sleep(stepDelay)
```

Comment:
The time.Sleep found after the transactions are mandatory as to be 100% sure the proposer received them. Remember that a node.SendTransaction will actually send it to self and others on go routines.

Reply:
there is a time.sleep(stepDelay) before calling propose block, to ensure transactions are put into pool.